src/server.js

```diff
@@ -17,6 +17,18 @@ server.get('/', (request, response) => {
 });
 
 server.post('/', (request, response) => {
+  // Don't log but give a very specific error. We don't want to fill the logs.
+  if (!isGithubRequestCrypographicallySafe(request, process.env.GITHUB_WEBHOOK_SECRET)) {
```

Comment:
This stuff looks great. Really appreciate you jumping in and helping us with this project. I have a few changes I'd recommend that should be made:

- Can you move this logic into the `github` function? The `success` param can become a `status` number to support a 500. The idea is that in the future we can swap out that function with `gitlab` or `bitbucket` with the only argument being `request.body` (which is exactly what `isGithubRequestCrypographicallySafe` needs).
- I also think this should be an assertion that throws. If anyone is sending malformed / request to this endpoint I would want my monitoring to alert me of the exception.